ZAPD/Globals.cpp

```diff
@@ -66,6 +66,22 @@ std::string Globals::FindSymbolSegRef(int32_t segNumber, uint32_t symbolAddress)
 	return "ERROR";
 }
 
+enum class ConfigType
+{
+	SYMBOL_MAP,
+	SEGMENT,
+	ACTOR_LIST,
+	OBJECT_LIST,
+	TEXTURE_POOL,
+	BG_CONFIG
+};
+
+const static std::map<std::string, ConfigType> ConfigTypeDictionary = {
```

Comment:
While `const static` is valid C++, `static const` is the usual order in which those two declaration specifiers are written. (And C goes even further by deprecating the use of storage-class specifiers other than at the beginning of a declaration.)

`const static` is valid, but it looks weird to anybody who's familiar with C or C++ (kind of like how there is a specific order for adjectives in English...) :)

Reply:
`<map>` should be included, otherwise this is relying on an indirect include and will fail to compile if that include disappears for whatever reason in the future.

Reply:
Oh, I didn't think it mattered... I thought these were the same type of thing: to me `const` just means "put in rodata" and `static` means "only usable in this file", or "put it in data and not on the stack", which sound like the same category to me.

Reply:
And yeah, I see what you mean with the include... I'm used to decomp where basically everything's included in every file.